consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGatewayTest.java

```diff
@@ -72,7 +72,8 @@ void shouldInvokeBreathingSpace() throws Exception {
         verify(breathingSpace).breathingSpace(anyString(), anyString(), any(BreathingSpaceRequest.class));
         verify(xmlToObject).convertXmlToObject(anyString(), any());
         verify(individualRequest).getRequestPayload();
-        verify(individualRequest).setRequestStatus(ProcessingStatus.PROCESSED.name());
```

Comment:
I think it's valid to keep this in as the CMCConsumerGateway.individualRequest() method will set the request status for a BreathingSpace request (see https://github.com/hmcts/civil-sdt/blob/9b4e649317f62ce7651d92ecf5f68ce0d9396f89/consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGateway.java#L64)

Reply:
done